maxim/logger/livekit/module_functions.py

```
@@ -0,0 +1,109 @@
+                    try:
+                        result = await original_func(*args, **kwargs)
+                        return result
+                    finally:
+                        intercept_execute_tools_task_result(
+                            result, start_timestamp, *args, **kwargs
+                        )
+
+                setattr(module, "_execute_tools_task", async_wrapper)
+
+                @functools.wraps(original_func)
+                def sync_wrapper(*args, **kwargs):
```

Ruff (0.11.9)

Missing return type annotation for private function `sync_wrapper`

(ANN202)